cpp/src/decisiontree/decisiontree.cu

```diff
@@ -112,7 +115,7 @@ void print(const DecisionTreeParams params) {
   CUML_LOG_DEBUG("max_features: %f", params.max_features);
   CUML_LOG_DEBUG("n_bins: %d", params.n_bins);
   CUML_LOG_DEBUG("split_algo: %d", params.split_algo);
-  CUML_LOG_DEBUG("min_rows_per_node: %d", params.min_rows_per_node);
+  CUML_LOG_DEBUG("min_samples_leaf: %d", params.min_samples_leaf);
```

Comment:
can you also add a debug statement for `min_samples_split` too?

Reply:
Done